### PR TITLE
[PLAT-2272] update java

### PR DESCRIPTION
**Pull request by ukayani**
- Update to official openjdk image (same as jemalloc branch).
- Use java 11 JRE
- Update data dog agent (supports java 12)
- Update yourkit agent (also with java 12 support)
- update jemalloc version

### CODE REVIEW

**Comment by ukayani** [comment]
I guess the name of the repo with `alpine` in it doesn't really suffice for the jemalloc branch..

File: README.md [hunk 1]
```diff
@@ -10,4 +10,42 @@ $>  ./publish.sh <version>
 
 The version generally looks like `v1.16.0`, or similar.
 
-## Current DataDog Agent Version: 0.24.1
+# Installed
+
+- **Java** - 1.8.x
```
Comment (nearlyfreeapps) [inline README.md]: Should this be updated now?

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,17 @@
 #Extending image to add useful binaries that help with debugging
-FROM lwieske/java-8:server-jre-8u172-slim
+FROM openjdk:11-jre-slim
```
Comment (nearlyfreeapps) [inline Dockerfile]: Only downside with using the jre image over the jdk is that it lacks `jcmd` which we have found to be useful when investigating memory related things in the past.

**Comment by nearlyfreeapps** [comment]
@ukayani Good call on the repo name. I have changed it to remove the alpine reference.

File: README.md (hunk 1 above)
Comment (ukayani) [reply README.md]: good catch

File: Dockerfile (hunk 2 above)
Comment (ukayani) [reply Dockerfile]: ah. I can switch it over to the jdk image. One thing id to eventually move away from though is having additional tools installed on the containers. Its nice to have but its kind of like a crutch. We should have enough observability into the service to not need to drop into containers. Its still a pipe dream though.

Comment (nearlyfreeapps) [reply Dockerfile]: Agreed its definitely a crutch and something I'd like to avoid as well. It's definitely not something that we have used often.

**Pull request merged**